tests/test_election.py

```diff
@@ -44,9 +45,9 @@ def test_timetable_sort_order():
     assert len(election.timetable) == 4
 
     assert election.timetable == [
-        {"label": "List of candidates published", "date": date(2021, 4, 9)},
         {"label": "Register to vote deadline", "date": date(2021, 4, 19)},
         {"label": "Postal vote application deadline", "date": date(2021, 4, 20)},
+        {"label": "List of candidates published", "date": date(2021, 4, 23)},
```

Comment:
Do you know why this date has changed? I think the 9th should be the right date for SOPNs in this case

Reply:
Not sure but have since sorted. This is ready for another look when you're ready. I've installed locally without failures, but it would be useful to know if I should add tests in ec-api.